### DISCUSSION TITLE
Hello, below is a proposal put forth by the team to update some of the values for animal XP and drops. This is aimed at providing a simpler solution to increase player engagement with the animal mechanic.

### DISCUSSION THREAD

**Discussion by Kartika70** [sunflower-land/sunflower-land#4946] [Ideas]
Hello, below is a proposal put forth by the team to update some of the values for animal XP and drops. This is aimed at providing a simpler solution to increase player engagement with the animal mechanic.

### Changes to Animal XP:

Sheep XP will convert similarly to the Chicken's.
lvl 1 - 60
lvl 2 - 120
lvl 3 - 240
lvl 4 - 360
lvl 5 - 480
lvl 6 - 660
lvl 7 - 840
lvl 8 - 1020
lvl 9 - 1200
lvl 10 - 1440
lvl 11 - 1680
lvl 12 - 1920
lvl 13 - 2160
lvl 14 - 2400
lvl 15 - 2700

Cow XP will convert to Sheeps'.
lvl 1 - 120
lvl 2 - 240
lvl 3 - 480
lvl 4 - 720
lvl 5 - 960
lvl 6 - 1320
lvl 7 - 1680
lvl 8 - 2040
lvl 9 - 2400
lvl 10 - 2880
lvl 11 - 3360
lvl 12 - 3840
lvl 13 - 4320
lvl 14 - 4800
lvl 15 - 5400

### Changes to Animal yield:

Chicken.
level 1: 1 egg
Level 2: 1 egg 
Level 3: 1 egg 1 feather
Level 4: 1 egg 1 feather
Level 5: 1 egg 1 feather
Level 6: 2 egg
Level 7: 2 egg 1 feather
Level 8: 2 egg 1 feather
Level 9: 2 egg 2 feather
Level 10: 2 egg 2 feather 
Level 11: 3 egg
Level 12: 3 egg 1 feather
Level 13: 3 egg 2 feather
Level 14: 3 egg 2 feather
Level 15 3 egg 3 feather
Level 15+: repeat level 15 rewards

### Changes to bounties:

More bounties will be available at base AND per upgrade.
10 bounties available at base and +5 per upgrade. Meaning a fully upgrade hen house (without boosts) would host 20 chickens and have 20 bounties for the week.

_Edited some values below_
Hen House:
> 10 bounties at base:
> 6 tickets, 4 coins

> 15 at 1st upgrade:
> 6 tickets, 9 coins

> 20 at 2nd upgrade:
>  6 tickets, 2 gems, 12 coins

Barn:
> 10 bounties at base:
> 6 tickets, 4 coins
﻿
> 15 at 1st upgrade:
> 6 tickets, 9 coins
﻿
> 20 at 2nd upgrade:
>  6 tickets (3 cow, 3 sheep), 2 gems (1 cow, 1 sheep), 12 coins (6 cow, 6 sheep)

IMPORTANT TO NOTE: These changes will not go live until auction week is finished.. so to not impact the seasonal pvp race mid-way through it.

I'd personally still like to change more in regards to this mechanic, but these can be some quick wins the team can do in a short amount of time. Also keep in mind, Animal Skills will be coming too..

We can also change Medicine recipe to replace iron with honey or other radish instead of eggs.. this is open for discussion too

Please be civil and respectful, note that values may change

_Originally posted by @TheDcol in https://github.com/sunflower-land/sunflower-land/discussions/4866_

**Comment by Kartika70** [comment]
I have a comment about chicken medicine. When a chicken reaches a certain level, all chickens ask for medicine.
The medicine is so expensive that it is not worth dealing with chickens. I don't think that is the goal. We should rethink the way chickens are medicated. If it is so expensive, then one medicine is enough for 10 chickens, or the cost of making the medicine should be drastically reduced.

**Comment by eliasSFL** [comment]
@Kartika70 I've converted this into a discussion

**Reply by Kartika70** [reply]
Thanks, I see the change proposal, but unfortunately this solution does not solve the problem I raised. (The Medicine recipe can also be modified to replace the iron with honey or other radishes instead of eggs.. this is also a matter of debate) Because even then you have to buy 10 mecinia for 10 chickens and the production in this case is not worth it. It also requires a lot of and expensive crops to feed the chicken, plus the cost of medicine, which I find disproportionate. Why isn't there a solution where you only have to buy 1 medicine for 10 chickens? The medicine should be more expensive, but you only have to buy it once. Or every time you refill the crop purchase, it also contains x amount of medicine, or any other solution that makes the situation easier would be nice.

2.Another topic 
Another comment: I have already indicated several times on the discord thread, it often happens that in the daily sunflower field codex you can't do the daily achievement, and you would skip that achievement. BUT no matter how much I set the Skip order, it returns the same result for days, often 1 week. so no matter how much I change the achievement, it is impossible.

Thank you.
